Database_Restructure/DB_restructure_documentation.md

```diff
@@ -0,0 +1,129 @@
+
+The DS team proposed a solution to separate the responsibilities of the two databases so as to remove the duplications while maintaining data security. This solution is stated in the "Purpose" section above.
+
+The first step taken by the DS team was to mirror all of the non-authentication tables from the Postgres database into the MongoDB database (as collections). Only the `schema.py` file in the DS codebase was altered in this process; no data generators were created for the new fields, and no changes were made to the API to reflect the additions. It was expected that as the restructure design was created and implemented in the codebase, the related collections mirrored from Postgres would be deleted.
+
+The DS team then created a Google spreadsheet as a brainstorming platform for consolidating non-authentication-related fields from the relationship-based Postgres tables into the document-based MongoDB collections (restructuring_research.xls).
+
+At the time, there had been a sizeable influx of DS students into the project with few Web students to complement. The current Web students on the project were occupied with priority web-related features, resulting in little Frontend/Backend input in restructuring the database. As a result, the DS team resorted to their own research, deciphering the frontend/backend codebases themselves, and making their best educated guesses as to how the Web team would access the data, and consequently, how to consolidate the Postgres tables/fields into MongoDB collections.
+
+The first round of redesign was completed in the Google spreadsheet. From there, the DS team needed an approach to deconstructing this design into manageable sections of work consistent with AGILE development, while maintaining enough independence in the work to avoid disrupting the effectiveness of the Web team. Their solution was to approach changes to the codebase 1-2 collections at a time:
+
+1. Alter or add the collection(s) to the `schema.py` file in the DS codebase (including the respective update collection(s))
+2. Make the respective changes to the data generators for the altered collection(s) in the `generators.py` file in the DS codebase
+3. Ensure the new data generators will operate with the `seeds.py` file in the DS codebase
+4. Ensure that the changes made in the previous 3 steps will operate with the DS API calls in the `api.py` file in the DS codebase
+
+With these steps completed, the DS team would notify the Web team of the changes, and the Web team would make the necessary alterations to the forms in the frontend and to the data pipeline in the backend.
+
+**NOTE!** This current process is insufficient and will need to be changed to ease the development process for both DS and Web - to reiterate, it was developed by the DS team in near-total absence of participation by the Web team. **It is highly recommended that moving forward, DS and Web both have specific members dedicated to the restructuring.**
```

Comment:
This documentation is sufficiently descriptive without being verbose. I like the recommendations for moving forward.